ipn/ipnlocal/local.go

```diff
@@ -463,10 +464,18 @@ func (b *LocalBackend) setClientStatus(st controlclient.Status) {
 		return
 	}
 
+	keyExpired := st.NetMap != nil && !st.NetMap.Expiry.IsZero() && time.Until(st.NetMap.Expiry) <= 0
```

Comment:
time.Until reads a bit awkward to me at least here. Maybe `time.Now().After(st.NetMap.Expiry)` ?

Reply:
Added an explicit check for NetMap and a comment to improve readability.